nbs/02_match_class.ipynb

```diff
@@ -224,7 +224,9 @@
     "        self.graph: DiGraph = graph\n",
```

Comment:
<p>The match class is simply a container for a match with the interface to access the actual graph through the logical vars.</p><p> </p><p>The logic for looking for matches should be in nb 003.</p><p> </p><p>The logic for parsing the string into the datastructure you want should be in the parsing, so I you want the parse to give you a set of vars for each phase, do that when parsing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>On it, thank you :)</p>